### PR TITLE
Try to cope with initial unresponsiveness of ssh

### PR DESCRIPTION
**Pull request by rmohr**
Retry establishing a connection to the VM a few times when we try to
connect to the VM the first time.

This may resolve some flakes which manifest themselves with

```
binding device to vfio driver failed
```

coming from the first script which gets executed.

### CODE REVIEW

**Comment by rmohr** [comment]
/cc @ormergi 
/cc @qinqon 
/cc @vladikr 

I have at least right now no better idea. Maybe you have?

File: cluster-provision/gocli/cmd/run.go [hunk 1]
```diff
@@ -502,6 +504,20 @@ func run(cmd *cobra.Command, args []string) (retErr error) {
...
+			err = _cmd(cli, nodeContainer(prefix, nodeName), "ssh.sh echo VM is up", "waiting for node to come up")
+			if err == nil {
+				break
+			}
+			logrus.WithError(err).Warningf("Could not establish a ssh connection to the VM, retrying ...")
+			time.Sleep(1 * time.Second)
+		}
+
+		if err != nil {
+			return fmt.Errorf("could not establish a connectdion to the node after a generous timeout: %v", err)
```
Comment (vladikr) [inline cluster-provision/gocli/cmd/run.go:518]: s/connectdion/connection

**Comment by vladikr** [comment]
Great!
/approve

**Comment by vladikr** [comment]
> I have at least right now no better idea. Maybe you have?

At least this will log the ssh failure now, so we will know the root cause

File: cluster-provision/gocli/cmd/run.go (hunk 1 above)
Comment (rmohr) [reply cluster-provision/gocli/cmd/run.go]: Done.

**Pull request merged**